jobs/rocoto/upp.sh

```diff
@@ -44,16 +44,11 @@ export PYTHONPATH
 export job="upp"
 export jobid="${job}.$$"
 
-###############################################################
-# shellcheck disable=SC2153,SC2001
-IFS='_' read -ra fhrs <<< "${FHRLST//f}" # strip off the 'f's convert to array
+export FORECAST_HOUR=$(( 10#${FHR3} ))
```

Comment:
The UPP also processes analysis for which FHR3 is `000` (This is needed in upp.x).
Just a note why there isn't the same logic as ATMOS_PRODUCTS here to handle the negative `-001`